tests/test_suite_generator/src/linux/ada_trinary/ada_trinary.adb

```diff
@@ -0,0 +1,34 @@
+with Ada.Text_IO;
```

Comment:
Make sure we have the standard copyright header on these ADA and C++ source files.